gcc/fold-const.c

```diff
@@ -1152,6 +1152,10 @@ const_binop (enum tree_code code, tree arg1, tree arg2)
       bool inexact;
       tree t, type;
 
+      /* For ftz-math disable all constant folding for now.  */
```

Comment:
The comment might be a bit inaccurate if you are disabling only FP constant folding, right?